package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
```

Comment:
Another idea: Can we install `cldr-data` as part of the compile-globalize grunt task? That'll make the task run a little longer, but avoid a lot of overhead for every other CI run and local install. Since we're committing the result to the repo, it seems pointless to install the dependency all the time.